src/private/remoteCamera.ts

```diff
@@ -0,0 +1,305 @@
+     */
+    available: boolean;
+    /**
+     * Either camera doesnt support to get state or It unable to apply command.
+     */
+    error: boolean;
+    /**
+     * Reset max out or already applied. Client Disable Reset.
+     */
+    reset: boolean;
+    /**
+     * ZoomIn maxed out.
+     */
+    zoomIn: boolean;
+    /**
+     * ZoomOut maxed out.
+     */
+    zoomOut: boolean;
+    /**
+     * PanLeft reached max left.
+     */
+    panLeft: boolean;
+    /**
+     * PanRight reached max right.
+     */
+    panRight: boolean;
+    /**
+     * TiltUp reached top.
+     */
+    tiltUp: boolean;
+    /**
+     * TiltDown reached bottom.
+     */
+    tiltDown: boolean;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate the reason for the error.
+   */
+  export enum ErrorReason {
+    CommandResetError,
+    CommandZoomInError,
+    CommandZoomOutError,
+    CommandPanLeftError,
+    CommandPanRightError,
+    CommandTiltUpError,
+    CommandTiltDownError,
+    SendDataError,
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to indicate the status of the current session.
+   */
+  export interface SessionStatus {
+    /**
+     * Whether the far-end user is controlling a  device.
+     */
+    inControl: boolean;
+    /**
+     * Reason the  control session was terminated.
+     */
+    terminatedReason?: SessionTerminatedReason;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate the reason the session was terminated.
+   */
+  export enum SessionTerminatedReason {
+    None,
+    ControlDenied,
+    ControlNoResponse,
+    ControlBusy,
+    AckTimeout,
+    ControlTerminated,
+    ControllerTerminated,
+    DataChannelError,
+    ControllerCancelled,
+    ControlDisabled,
+  }
+
+  GlobalVars.handlers['remoteCamera.capableParticipantsChange'] = handleCapableParticipantsChange;
+  GlobalVars.handlers['remoteCamera.handlerError'] = handleHandlerError;
+  GlobalVars.handlers['remoteCamera.deviceStateChange'] = handleDeviceStateChange;
+  GlobalVars.handlers['remoteCamera.sessionStatusChange'] = handleSessionStatusChange;
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Fetch a list of the participants with controllable-cameras in a meeting.
+   * @param callback Callback contains 2 parameters, error and participants.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * participants can either contain an array of Participant objects, incase of a successful fetch or null when it fails
+   * participants: object that contains an array of participants with controllable-cameras
+   */
+  export function getCapableParticipants(
+    callback: (error: SdkError | null, participants: Participant[] | null) => void,
+  ): void {
+    if (!callback) {
+      throw new Error('[remoteCamera.getCapableParticipants] Callback cannot be null');
+    }
+    ensureInitialized();
```

Comment:
It's probably best to pass `FrameContexts.sidePanel` to the `ensureInitialized();` call since these APIs won't work from any other context.